src/openssl/lib_crypto.cr

```diff
@@ -15,7 +21,13 @@
   end
 {% end %}
 
-@[Link(ldflags: "`command -v pkg-config > /dev/null && pkg-config --libs --silence-errors libcrypto || printf %s '-lcrypto'`")]
+{% if flag?(:win32) %}
+  @[Link("crypto")]
+  @[Link("crypt32")] # CertOpenStore, ...
+  @[Link("user32")]  # GetProcessWindowStation, GetUserObjectInformationW, _MessageBoxW
```

Comment:
Linking here seems unfortunate, particularly for `_MessageBoxW` :/
But seems like it couldn't be declared within OpenSSL itself? 
https://github.com/openssl/openssl/blob/3e3ad3c54855dd534437871a0c78858de5e3d246/NOTES-WINDOWS.md?plain=1#L207